### DISCUSSION TITLE
How can I enable instant answering modules on docker?

### DISCUSSION THREAD

**Discussion by IcyAlmond** [searxng/searxng#1120] [Q&A] [answered]
How can i enable the instant answering modules? I'm on the docker container and the generated settings.yml doesn't have it commented anywhere so i don't know the syntax. I want to enable the random value generator specifically

**Comment by IcyAlmond** [comment, marked as answer]
it's already enabled looks like but can't find an option to toggle

**Reply by return42** [reply]
if you have questions join us on Matrix  [#searxng:matrix.org](https://matrix.to/#/#searxng:matrix.org)
